frontend/src/components/ForgotPassword/styles.ts

```diff
@@ -2,7 +2,7 @@ import { Box } from '@mui/material';
 import { styled } from '@mui/material/styles';
 
 export const RegistrationFormWrapper = styled(Box)`
-  height: 83vh;
+  height: 86vh;
   display: flex;
   justify-content: center;
```

Comment:
you shouldn't set magic numbers in production, and global styling must be improved in the future

Reply:
yes - this is fix only before demo)